packages/listbox/src/ListboxMixin.js

```diff
@@ -277,6 +281,20 @@ const ListboxMixinImplementation = superclass =>
       this.__teardownEventListeners();
     }
 
+    resetInteractionState() {
+      super.resetInteractionState();
+      this.submitted = false;
+    }
+
+    reset() {
+      this.formElements.forEach(formElement => {
+        // eslint-disable-next-line no-param-reassign
+        formElement.checked = formElement.choiceValue === this._initialModelValue;
```

Comment:
would that not put a `checked` property an every form element? even normal inputs? 🤔

Reply:
Normal inputs can't be a child of the listbox (ChoiceGroupMixin will throw an error).
Checkbox and radio can be placed inside the listbox, but this breaks the whole listbox, but i don't think we have to guard this, and especially not within the reset function.